doc/syntax.rst

```diff
@@ -83,10 +83,6 @@ a wide array of quantum hardware:
 * Statically typed - you must declare the variable type, and variables
   and arguments of conflicting types are **not** automatically cast to the correct type.
 
-.. 
-
-* Array variables may be declared, but Blackbird does not support array manipulation.
```

Comment:
I suppose this is still somewhat true, since the arrays do not support element-wise or matrix processing? E.g., element wise multiplication or matrix multiplication. Also, you can't do something like `sin(an_array)` (can you?)

Perhaps it could be updated, with "Array variables may be declared, and individual elements accessed through indexing, but Blackbird does not support array manipulation."

Reply:
Sure, that's true. I'll update it!